telebot34.py

```diff
@@ -0,0 +1,242 @@
+import os
+import csv
+import telebot
+from telebot import types
+from datetime import datetime
+
+API_TOKEN = ""
+
+bot = telebot.TeleBot(API_TOKEN)
+DATE_FORMAT = "%d.%m.%Y"
+users = {}
+
+
+def is_valid_name_surname_action(name_surname_action):
+    return not (" " in name_surname_action or len(name_surname_action) < 2)
+
+
+@bot.message_handler(content_types=["text"])
+def start(message):
+    user_id = message.from_user.id
+    if message.text == "Регистрация":
+        # create empty user
+        users[user_id] = {}
+        remove_initial_keyboard(user_id, "Как тебя зовут?")
+        bot.register_next_step_handler(message, get_name)
+    elif message.text == "TODO":
+        users[user_id] = {}
+        remove_initial_keyboard(user_id, "Чтобы бы  ты хотел запланировать ?)")
+        bot.register_next_step_handler(message, get_action)
+    elif message.text == "Планы сегодня":
+        users[user_id] = {}
+        question = f"{user_id}, хотите узнать что на сегодня запланировано ?"
+        render_yes_now_keyboard(user_id, question, "plan")
+    else:
+        render_initial_keyboard(user_id)
+
+
+@bot.callback_query_handler(func=lambda call: call.data.startswith("plan_"))
+def callback_worker2(call):
+    user_id = call.from_user.id
+    if call.data == "plan_yes":
+        now = datetime.now()
+        wnow = now.strftime("%d.%m.%Y")
+        user_actions = []
+        csv_dir = os.path.join("test_files", "csv")
+        file_path = os.path.join(csv_dir, "employ6.csv")
+
+        with open(file_path) as csv_file:
+            csv_reader = csv.DictReader(csv_file)
+            for row in csv_reader:
+                if not row["id"] == str(user_id):
+                    continue
+                actdate = row["date"]
+                actdatee = datetime.strptime(actdate, "%d.%m.%Y")
+                actiondate = actdatee.strftime("%d.%m.%Y")
+                if wnow == actiondate:
+                    user_actions.append(row["action"])
```

Comment:
```suggestion
                actdate = row["date"]
                if wnow == actdate:
                    user_actions.append(row["action"])
```
Если в файле дата в неправильном формате, то `strptime` вернет ошибку. А если в правильном, то нет смысла туда-назад конвертировать